Sources/EthWrapper/EthWrapper.swift

```diff
@@ -91,10 +91,10 @@ public class EthWrapper: BleWrapper {
         }, failure: failure)
     }
     
-    public func signTransaction(path: String, rawTxHex: String, resolution: LedgerEthTransactionResolution?, success: @escaping DictionaryResponse, failure: @escaping ErrorResponse) {
+    public func signTransaction(path: String, rawTxHex: String, option: LedgerEthTransactionResolution?, success: @escaping DictionaryResponse, failure: @escaping ErrorResponse) {
```

Comment:
This is the app you are wrapping and where you are dispatching to https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/hw-app-eth/src/Eth.ts

You can see that signTransaction has a different input, you need to add support for ` async clearSignTransaction(`

The original signTransaction method should not be modified at all.